urllib3/connection.py

```diff
@@ -56,7 +56,7 @@ class ConnectionError(Exception):
     'https': 443,
 }
 
-RECENT_DATE = datetime.date(2014, 1, 1)
+RECENT_DATE = datetime.date(2016, 1, 1)
```

Comment:
Can we get a comment above here that explains our "two-year" rule? What is the latest this should be? What is the earliest it should be?